sda-commons-server-spring-data-mongo/src/test/java/org/sdase/commons/server/spring/data/mongo/SpringDataMongoBundleCustomConverterIT.java

```diff
@@ -34,7 +34,7 @@ class MorphiaBundleCustomConverterIT {
   @Order(1)
   static final DropwizardAppExtension<MyConfiguration> DW =
       new DropwizardAppExtension<>(
-          MorphiaTestApp.class,
+          TestApp.class,
```

Comment:
May we change the variable name on line 60, as well?

```
Person johnDoeFromMorphia =
        mongoOperations.findAll(Person.class).stream()
            .filter(p -> p.getId().equals(johnDoe.getId()))
            .findFirst()
            .orElse(null);
    assertThat(johnDoeFromMorphia).isNotNull();
    assertThat(johnDoeFromMorphia.getPhoneNumber())
        .extracting(PhoneNumber::getCountryCode, PhoneNumber::getAreaCode, PhoneNumber::getNumber)
        .containsExactly("+49", "172", "123456789");
```

Reply:
Fixed 👍